Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1140,5 +1140,6 @@
     <string name="test_api_body_other_keys">"Other key"</string>
     <!-- NOTR -->
     <string name="test_api_calculate_risk_level">"Calculate Risk Level"</string>
+    <string name="test_api_country_filter_status">Country filter applied for: \n%1$s</string>
```

Comment:
Please only add strings to values-de/strings.xml. And add the corresponding comments for the translation system. In this case it would be _<!-- NOTR -->_.

As that is not enough you also have to add only a placeholder string to values/strings.xml. 
`<string name="test_api_country_filter_status" />`

Please remove the strings for all other languages. To clarify this process I'll send you the link to our internal wiki.

Reply:
Also the other string is missing `test_api_country_settings_heading`.